### PR TITLE
I added a proper introduction to the 'Introduction to the Coding Inte…

### PR DESCRIPTION
**Pull request by ol-web**
…rview Take Home Projects' section

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

first PR ever btw 
let me know if everything is nice and dandy 👍

### CODE REVIEW

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-This introduction is a stub
+When applying for a job as a developer, there's a good chance the hiring team will give you something called a "take-home project". As the name suggests, it's something that you're supposed to do on your own, without the interviewer looking over your shoulder. Unlike other parts of the coding interview, which often focus more on theoretical knowledge and less on actual app development - a take-home project is usually the best place to show your skills in practice.
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+In this section you'll be able to work on such projects!
```
Comment (co16353sidak) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: possibly change this to -> `able to work on examples of such projects`
and add `practice on these to get an idea of how to work on such projects.`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@oL-web Good Work on your first PR.

Please review some changes I have requested that may improve the readability more.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 1 above)
Comment (oL-web) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Added the first line but the second one seems redundant. It repeats the words "practice" and "such projects" as well.

**Comment by oL-web** [comment]
> @oL-web Good Work on your first PR.
> 
> Please review some changes I have requested that may improve the readability more.

**Review by thecodingaviator — APPROVED** [review]
LGTM!

**Review by theNaut1lus — APPROVED** [review]
@oL-web LGTM, changes have been made.

@RandellDawson please merge this PR

**Comment by RandellDawson** [comment]
Congrats @oL-web  on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**